keras/saving/hdf5_format.py

```diff
@@ -615,7 +615,27 @@ def load_optimizer_weights_from_hdf5_group(hdf5_group):
   return [weights_group[weight_name] for weight_name in optimizer_weight_names]
 
 
-def save_weights_to_hdf5_group(f, layers):
+def save_layer_weights_to_hdf5_group(f, layer):
```

Comment:
Presumably you're going to want to save a set of weights, not a layer. Because you need to be able to select the weights.

Probably:
`def save_weights_subset_to_hdf5_group(f, weights)`